forge-game/src/main/java/forge/game/ability/effects/RollPlanarDiceEffect.java

```diff
@@ -19,17 +18,11 @@ public class RollPlanarDiceEffect extends SpellAbilityEffect {
      */
     @Override
     public void resolve(SpellAbility sa) {
-        boolean countedTowardsCost = !sa.hasParam("NotCountedTowardsCost");
```

Comment:
don't the rules need this and _Fractured Powerstone_ was missing it?

Reply:
Fixed it